docker-compose.yml

```diff
@@ -77,11 +77,13 @@ services:
       - mysqlvolume:/var/lib/mysql
 
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.10
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.7.1
     environment:
       - discovery.type=single-node
       - xpack.security.enabled=false
       - ES_JAVA_OPTS=-Xms512m -Xmx512m
+    ports:
+      - "9200:9200"
```

Comment:
This is a bit controversial. I first added it just so I can poke around inside ES from my browser on the host. But I think that's likely to happen again so I decided to leave it. It would cause trouble, in local dev, if you already have an ES running in another terminal. But that's no different from using `:8000` in another terminal too.